src/loaders/FileLoader.js

```diff
@@ -168,20 +168,20 @@ Object.assign( FileLoader.prototype, {
 
 			request.addEventListener( 'load', function ( event ) {
 
-				var response = this.response;
+				var response = event.target.response;
```

Comment:
If what you propose is correct, how about this instead:
```javascript
var target = event.target;
```
...and then make the remaining necessary changes.

Reply:
done